docs/getstarted/get_started_jupyter.md

```diff
@@ -0,0 +1,120 @@
+
+```bash
+$ ls
+$ README.md mnist.ipynb
+```
+
+In this guide, we will use the `mnist_cnn.ipynb` Jupyter Notebook.
+
+### Initialize new project
+Now that we have the code, we want to associate this directory with the new project you just created on FloydHub. Ensure that you are inside the `quick-start-pytorch` directory and execute:
+
+```bash
+$ floyd init mnist-pytorch
+Project "mnist-pytorch" initialized in the current directory
+```
+
+This tells Floyd that all the jobs run from this directory belong to the same project.
+
+## Running Jupyter Notebook on FloydHub
+
+Starting a Jupyter Notebook on FloydHub is very simple. Use the [floyd run](../commands/run.md) command with `--mode jupyter` flag.
+
+Execute the following command from the command line:
+
+```bash
+$ floyd run --mode jupyter --gpu --env pytorch-0.2
+Creating project run. Total upload size: 21.9KiB
+Syncing code ...
+[================================] 23333/23333 - 00:00:00
+NAME
+--------------------
+saip/mnist-pytorch/3
+
+Setting up your instance and waiting for Jupyter notebook to become available ..............
+
+Path to jupyter notebook: https://www.floydhub.com/notebooks/pCoPyzZtYeo6mE9PpSWsmY
+```
+
+This will take a little bit. As it executes, Floyd is doing the following behind the scenes:
+
+- Sync your local code to FloydHub's server
+- Provision a GPU instance on the cloud (if you want CPU, drop the `--gpu` flag)
+- Set up an deep learning environment with PyTorch installed (because `--env pytorch`)
+- Start a Jupyter server on the cloud, and open the url in your browser
+
+You can also open the link to the your Jupyter dashboard using the displayed URL. For example:
+
+![Jupyter](../img/pytorch_notebook.png)
+
+Open the `mnist.ipynb` Notebook and start training your model interactively!
+
+## Next steps
+
+### Check the status of your job
+
+You can view the status of your job from your terminal using the [floyd status](../commands/status.md) command
+
+```bash
+$ floyd status saip/mnist-pytorch/3
+JOB NAME                CREATED         STATUS      DURATION(s)  INSTANCE    DESCRIPTION
+----------------------  --------------  --------  -------------  ----------  -------------
+saip/mnist-pytorch/3    16 minutes ago  running               0  gpu
+```
+
+You can also view the status by going to the project page in the web dashboard.
+
+### Stopping your Notebook
+
+!!! warning
+    Jupyter Notebooks are designed for interactive development. Your job starts running on FloydHub's server when you execute the `floyd run --mode jupyter` command and it continues to be active till you explicitly stop your job.
+
+    Hence, even if you are not actively executing code inside your Notebook, the Jupyter server is still active on FloydHub and you are billed for the time.
+
+To stop your notebook you can use the [floyd stop](../commands/stop.md) command.
+
+```bash
+$ floyd stop saip/mnist-pytorch/3
+Experiment shutdown request submitted. Check status to confirm shutdown
```

Comment:
This code snippet is rendered a little weird 

![image](https://user-images.githubusercontent.com/6779603/31152010-fa10b582-a84e-11e7-98e2-fc5f93317f65.png)